Docs: replacing "setting" to "option" in world docs (#2622)
ArchipelagoMW/Archipelago: worlds/witness/docs/en_The Witness.md

Alternatively (or additionally), you can play "Door shuffle", where some doors won't open until you receive their "key".

> Receiving lasers as items is also a possible option.

Reviewer: Oooo I don't like this, this massively overlaps with the colloquial definition of "option"
Maybe this can be reworded to something like "is also an option you can choose"?

(Tbh this is part of the reason I would've preferred "settings" as the new catch-all term, but eh)

Author: ```suggestion
You can also chose to receive lasers as items as an option.
``` how does this sound?

Reviewer: Sure, either of those work for me, I probably thing Rever's is better? Otherwise someone might think they can do some in-game action for it or sth

Author: Scipio caught a typo tho o:

> You can also choose to receive lasers as items as an option.

Author: ```suggestion
You can also set lasers to be items you can receive.
```

Reviewer 2: I like that second one better
